3473/0/about_dice_project.rb

```diff
@@ -1,12 +1,13 @@
 require File.expand_path(File.dirname(__FILE__) + '/neo')
 
 # Implement a DiceSet Class here:
-#
+# :reek:Attribute, :reek:InstanceVariableAssumption
+# :reek:TooManyStatements
 class DiceSet
-  attr_reader :values
-
-  def roll(count)
-    @values = Array.new(count).collect! { |_item| Random.new.rand(1..6) }
+  attr_accessor :values
+  def roll(size)
+    @values = []
```

Comment:
why did you replace Array initialization with times?

Reply:
I did not replace. The previous version belongs to #3501